recipe/meta.yaml

```diff
@@ -49,12 +49,14 @@ outputs:
         - cmake
         - ninja
       host:
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
```

Comment:
This is needed to align `requirements/build` and `requirements/host` on the same CUDA version